### ISSUE TITLE
FHIRFormatError in FHIR Medication endpoint

### ISSUE THREAD

**Issue by ivandivito** [IHTSDO/snowstorm#132]
When i try to make a request to the Fhir Medication endpoint the following error arise.

```
{
    "resourceType": "OperationOutcome",
    "issue": [
        {
            "severity": "error",
            "code": "processing",
            "diagnostics": "Failed to call access method: java.lang.RuntimeException: org.hl7.fhir.exceptions.FHIRFormatError: unexpected non-end of element null::div  at line 1 column 20"
        }
    ]
}
```

I tested it in the public server with an insulin concep with id 781942001 (https://snowstorm-fhir.snomedtools.org/fhir/Medication/781942001) but the problem persist with all the medication codes i tested.

Thank for all your work on this project.

**Comment by kaicode** [comment]
I confirm that the request is failing in an unexpected way and that the concept id provided is valid.
`781942001 | Product containing precisely insulin lispro 100 unit/1 milliliter conventional release solution for injection (clinical drug) |`
@pgwilliams Could you take a look please?

**Comment by pgwilliams** [comment]
Fix now available in the develop branch.  As per our agreed process, this issue will remain open until a release has been cut from the master branch.   Thank you for reporting that @ivandivito

**Comment by kaicode** [comment]
Fixed in [5.0.6](https://github.com/IHTSDO/snowstorm/releases/tag/5.0.6).
